core/eth/tx.go

```diff
@@ -395,11 +398,13 @@ func (t *Transactor) GetOperatorStakesForQuorums(ctx context.Context, quorums []
 		return nil, err
 	}
 
-	state := make([][]core.OperatorStake, len(state_))
+	state := make(core.OperatorStakes, len(state_))
 	for i := range state_ {
-		state[i] = make([]core.OperatorStake, len(state_[i]))
+		quorumID := core.QuorumID(i)
```

Comment:
I don't think this is correct; see my comment above. Should probably be using `quorumID = quorum[i]`.

Reply:
Done